src/HttpLib/Test/PseudoTCPMessageTest.java

```diff
@@ -0,0 +1,24 @@
+package HttpLib.Test;
+
+import HttpLib.protocol.UDP.PseudoTCPMessage;
+import HttpLib.protocol.UDP.PseudoTCPPacket;
+import org.junit.jupiter.api.Test;
+
+public class PseudoTCPMessageTest {
+
+    @Test
+    public void test_getPackets() {
+        // Arrange
+        int numPackets = 1013 * 15 + 5; // will create 16 packets
```

Comment:
I don't think this calculation is correct. I think each packet has a total size of 1024 bytes, with a payload of 1013.
So 1024*16.
Also, I realize that 1024 could be a static constant in the PseudoTCPPacket.